app/tests/setup.js

```diff
@@ -0,0 +1,22 @@
+require('babel-register') ({
+  presets: ['es2015', 'react', 'stage-0']
```

Comment:
I think it's okay to keep as-is currently. If we find that we need to do it later, we will tackle that in a separate PR.